packages/ui/src/utils/version-compare.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Compare the version of camel catalogs
+ *
+ * @param v1 The first version.
+ * @param v2 The second version.
+ * @returns The weight of the schema against the model.
```

Comment:
Could it be possible to explain the return of the function like we're doing for the tests? I think this description might not be 100% accurate :)

Reply:
That's an incorrect statement. Thanks for the catch!